OBAKitCore/Models/REST/References/Stop.swift

```diff
@@ -89,13 +89,22 @@ public class Stop: NSObject, Identifiable, Codable, HasReferences {
 
     /// The coordinates of the stop.
     public let location: CLLocation
-
     /// Identifies whether this stop represents a stop, station, or station entrance.
     public let locationType: StopLocationType
 
     /// A human-readable name for this stop.
     public let name: String
 
+    /// A list of route IDs served by this stop.
+    ///
+    /// This value is only decoded/encoded via Codable. When retrieving from GRDB, this value will be empty.
+    let routeIDs: [RouteID]
```

Comment:
are there circumstances where this being empty when retrieved from GRDB is going to be problematic?

Reply:
GRDB will return an empty array if I fetch for `ids: []`. Also, I actually do insert the `routeIDs` into GRDB, so I removed that comment.